### ISSUE TITLE
REQUEST: Widescreen support for the ever growing library of SNES games that support it

### ISSUE THREAD

**Issue by lonkelle** [snes9xgit/snes9x#749]
Would a this new technique ever be considered for snes9x to allow for widescreen SNES hacks? 

https://hackaday.com/2021/06/22/modifying-a-snes-rom-to-be-widescreen/

**Comment by qwertymodo** [comment]
This requires some serious reworking of the PPU in ways that are not at all accurate to the hardware.  Not only does this present a significant amount of work, the changes aren't actually standardized, so it would be difficult to make our changes compatible with the existing implementations.  I won't go so far as to flat-out say this won't ever happen, but it's HIGHLY unlikely.

**Comment by lonkelle** [comment]
So basically, do these two points some up what would make this feasible for snes9x?

• The widescreen technique needs to be more official / standardized / agreed upon so the snes9x team doesn’t have to redo their implementation as this new standard grows (and at this point it is immature)
• The tech should consider the fact that it needs to tell the emulator to “change into enhanced emulator-only PPU mode” (likely through the header).

**Comment by qwertymodo** [comment]
I don't believe the header is used for identification, but identification would be part of point 1 (i.e. the need for standardization).

**Comment by lonkelle** [comment]
Is there more to standardization than just identification? Because the “map mode” part of the SNES header could easily be amended to include “widescreen PPU enhanced” along with the other combination of bytes that specify the map mode. It even pseudo-fits given it would be specifying a special emulation-only “PPU Mapping Mode”. 

My goal is to basically bring this discussion up. If the issue is standardization then why don’t we did so? Its not true to the hardware so that’s one reason (tho neither are runahead frames), but it’s gotten enough popularity to merit putting more work into getting these widescreen hacks out there. If you can think of any more ways in which this tech should be standardized aside from telling the emulator it needs the special PPU mode - lemme know. ☺️ 

Obviously a separate PPU module would need to be created so the work is non-trivial but I feel like it deserves to at least be discussed given it is a cool technology for these old amazing games.

**Comment by p3st-textures** [comment]
just read this and i don´t think it is worth  the time tbh ,

cuz all real widescreen hacks we got since bsnes-hd released  2019 are 2  these where really early one and than nothing happens , 
no one really seems to care for it on  from the romhacking community

and the handfull games you can get kinda to work without sprites popping , aren´t hacks but more luck that they work somehow ^^   i allways search for new ones or at least projects but its an wastland  and not really ever growing 

just saying .
